client/src/components/ListDeleter.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+
+function ListDeleter({ onCancel }) {
+  return (
+    <Wrapper>
+      <h4>Do you want to delete this list?</h4>
+      <div>
+        <button onClick={() => onCancel(false)}>Cancel</button>
+        <button onClick={() => onCancel(false)}>Proceed</button>
+      </div>
+    </Wrapper>
+  );
+}
+
+export default ListDeleter;
+ListDeleter.propTypes = {
+  onCancel: PropTypes.func,
+};
+
+const Wrapper = styled.div`
+  width: 70%;
+  height: 20%;
+  padding: 0 15px;
+  background: #fff;
+  box-shadow: 3px 3px 6px rgba(0, 0, 0, 0.16);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+
+  @media only screen and (min-width: 600px) {
+    width: 30%;
+  }
+  h4 {
+    color: var(--font-title-welcome);
+  }
+  button {
+    background-color: transparent;
+    border: none;
+    margin-left: 10px;
+    font-size: 1rem;
+  }
+  > div {
+    align-self: flex-end;
+  }
+
+  button:last-of-type {
+    color: var(--font-title-welcome);
+  }
+`;
```

Comment:
1. in this case, would it make sense to put this into global styles, for the case you need to use this modal in another place? 
2. perhaps it would be better to use more relative values than px's for better responsiveness?

Reply:
1. This is the only place I will use it therefor no :) 

2. Will check but it´s already responsive :)